samples/SocketsSample/Startup.cs

```diff
@@ -58,10 +59,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseSignalR(routes =>
             {
-                routes.MapHub<DynamicChat>("/dynamic");
-                routes.MapHub<Chat>("/default");
-                routes.MapHub<Streaming>("/streaming");
-                routes.MapHub<HubTChat>("/hubT");
+                routes.MapHub<DynamicChat>("/dynamic", options => options.Transports = TransportType.LongPolling | TransportType.ServerSentEvents);
+                routes.MapHub<Chat>("/default", options => options.Transports = TransportType.LongPolling | TransportType.ServerSentEvents);
```

Comment:
Don’t ruin the sample..

Reply:
It's temporary, I should've mentioned that up at the top, but thanks for the reminder.

Reply:
😄